cmd/extproc/mainlib/main.go

```diff
@@ -19,12 +20,16 @@ import (
 	"github.com/envoyproxy/ai-gateway/internal/version"
 )
 
+const (
+	defaultAddress  = ":1063"
+	defaultLogLevel = "info"
+)
+
 var (
 	configPath = flag.String("configPath", "", "path to the configuration file. "+
 		"The file must be in YAML format specified in extprocconfig.Config type. The configuration file is watched for changes.")
-	// TODO: unix domain socket support.
-	extProcPort = flag.String("extProcPort", ":1063", "gRPC port for the external processor")
-	logLevel    = flag.String("logLevel", "info", "log level")
+	extProcAddr = flag.String("extProcAddr", defaultAddress, "gRPC address for the external processor")
```

Comment:
This could be a breaking change for users using the extproc directly. The controller doesn't explicitly set the flag and relies on the default value so that should not break.

Reply:
yeah